src/web/components/Card/Card.tsx

```diff
@@ -265,8 +269,10 @@ export const Card = ({
 								)}
 								<CardFooter
 									format={format}
+									badge={badge || undefined}
+									palette={palette}
 									age={
-										webPublicationDate ? (
+										webPublicationDate && !badge ? (
```

Comment:
If you look at the other examples of use cases being passed into `CardFooter` there's a pattern where the thing to show, (`age`, `mediaMeta` or `commentCount`) is passed in as a prop and then the `CardFooter` uses a `footer` tag with some flex position styling to wrap it (you might not need any flex styling here - not sure).

So we could have another prop of `badge` and that could be set to the react component that represents how the badge renders. That way we keep the logic consistent, we only extend it, and we don't need to use this `!badge &&` check here.

Shout if I'm not making sense - happy to step though it!